.github/workflows/ci-build.yaml

```diff
@@ -0,0 +1,48 @@
+name: CI
+on: [push, pull_request]
+
+jobs:
+  build-and-deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout source
+        uses: actions/checkout@v2
+
+      - name: Install graphviz
+        shell: bash -l {0}
+        run: |
+          sudo apt-get update && sudo apt-get install graphviz
```

Comment:
I think we can drop this as the `python-graphviz` conda package should install the system graphviz library

Reply:
Awesome!